dataplane/src/pipeline/test_utils.rs

```diff
@@ -6,16 +6,6 @@
 use crate::pipeline::sample_nfs::{BroadcastMacs, DecrementTtl, InspectHeaders, Passthrough};
 use crate::pipeline::{DynNetworkFunction, nf_dyn};
 use net::buffer::TestBuffer;
-use net::eth::Eth;
-use net::eth::ethtype::EthType;
-use net::eth::mac::{DestinationMac, Mac, SourceMac};
-use net::headers::{Headers, Net};
-use net::ipv4::Ipv4;
-use net::ipv4::addr::UnicastIpv4Addr;
-use net::packet::{InvalidPacket, Packet};
-use net::parse::DeParse;
```

Comment:
And that's how you know that `net` was the right place for that function :slightly_smiling_face: